contactus.html

```diff
@@ -77,7 +77,7 @@
         <li class="horizontal"><a href="http://systers.io/slack-systers-opensource/" target="_blank">Slack</a></li>
         <li class="horizontal"><a href="https://github.com/systers" target="_blank">GitHub</a></li>
         <li class="horizontal"><a href="https://twitter.com/systers_org" target="_blank">Twitter</a></li>
-        <li class="horizontal"><a href="http://systers.org/mailman/listinfo/systers-opensource" target="_blank">Systers Mailing List</a></li>
+        <li class="horizontal"><a href="http://systers.org/mailman/listinfo/systers-opensource" target="_blank">Systers Open Source mailing list</a></li>
```

Comment:
@isabelcosta let us be consistent with the capitalization and use capital `Mailing List` instead.

Reply:
Of course @sara-02 , I'll change that now